prefix-uniqueness.graphql

```diff
@@ -0,0 +1,23 @@
+schema
```

Comment:
EXAMPLE: The schemas in this document lack the meta-`@core`

Reply:
Fixed